ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-next__47.yaml

```diff
@@ -57,14 +57,6 @@ images:
       - destination_dir: .
         source_path: /go/bin/mtreceive_adapter
   to: knative-eventing-kafka-mtreceive-adapter
-- dockerfile_path: openshift/ci-operator/knative-images/post-install/Dockerfile
-  from: base
-  inputs:
-    bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/post-install
```

Comment:
we have the image now here: https://github.com/knative-sandbox/eventing-kafka/tree/main/config/channel/post-install

that's for the channel. I think we should add that too (via the generator).  

PR for source migrator is pending

Reply:
yes but this is a different version migrator. I can add that in a separate PR and I think we won't need an image for it. Just the manifest. In this PR I'm removing Travis post-install.

Reply:
I can see there's this image https://github.com/knative-sandbox/eventing-kafka/blob/main/config/channel/post-install/storage-version-migrator.yaml#L38 , okay. I thought there is a generally built one that we can directly address.

Ok, I'll add that new image then.

/hold

Reply:
> let's wait until there is the source- image... it's a bit tricky


HOWEVER. You are correct with removing _this_ image :smile: 

/unhold
/lgtm
/approve